modules/op-release-notes-1-14.adoc

```diff
@@ -314,3 +314,15 @@ With this update, {pipelines-title} General Availability (GA) 1.14.2 is availabl
 * Before this update, when you started a pipeline run using {pac}, {tekton-results} did not store information about this pipeline run. Because of this issue, the web console plugin did not include the pipeline run in the execution statistics display. With this update, {tekton-results} stores information about {pac} pipeline runs and these pipeline runs are included in the execution statistics display.
 
 * Before this update, when you started many pipeline runs using {pac} at the same time and these pipelines runs included a `max-keep-run` annotation, the {pac} watcher component failed to process some of the pending pipeline runs and they remained in a pending state. With this update, {pac} pipeline runs are processed correctly.
+
+[id="release-notes-1-14-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.14.3
+
+With this update, {pipelines-title} General Availability (GA) 1.14.3 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-14-3_{context}"]
+=== Fixed issues
+
+* Before this update, when you started many pipeline runs using {pac} at the same time and these pipelines runs included a `max-keep-run` annotation, the {pac} watcher was unable to reconcile the pipeline runs because of a race condition between deletion of existing pipeline runs and processing new pipeline runs. Because of this issue, some pipeline runs could not be processed.  With this update, the watcher processes pipeline runs.
```

Comment:
```suggestion
* Before this update, when you started many pipeline runs using {pac} at the same time and these pipelines runs included a `max-keep-run` annotation, the {pac} watcher was unable to reconcile the pipeline runs because of a race condition between deletion of existing pipeline runs and processing new pipeline runs. Because of this issue, some pipeline runs could not be processed.  With this update, the {pac} watcher processes pipeline runs.
```

Reply:
I wonder if there is a specific number of pipeline runs that would cause this issue, as opposed to "many"? I realize there may not be an absolute number,  I just feel "many" is a bit vague (but using that word may be unavoidable as well).

Reply:
Unfortunately this is unavoidable as I don't think there was testing for the exact threshold, the engineers concentrated on fixing the problem.